pattern/change-developers-mindset (#63)
gruetter/innerSourceTest: patterns/1-initial/change-the-developers-mindset.md

* There is already a successful InnerSource group in the early stages.
* Code visibility is product dependent.

## Forces  

* Managers are previous developers, so they like the way they were promoted and they want to proceed in similar ways.
* Managers restrict what developers can spend time on, perhaps because of KPIs they have to meet and their belief in the best way to meet those.
* The approach to the InnerSource initiative is top down - it is supported by top management.
* Different teams within the company decide whether and how to proceed with InnerSource.
* No formal training exists.
* Processes are not clearly defined.

## Solution  

* Show reward/recognition of the developer team that is engaged in InnerSource.
* Formalize training.
* Clarify more processes.
* Give middle management specific objectives to make InnerSource successful.
* Listen to manager complaints and fears and counter them.

## Resulting Context

* Software from several InnerSource projects is used across the several development teams.
* There is improved collaboration within the same developer team (mentorship and so on).

## Known Instances

TBD

## Status  

* Initial

## See Also

> * For further improvements of this pattern, please als review these [comments from the ISC Fall Summit 2017](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/63#pullrequestreview-67393334).

Reviewer: ```suggestion
* For improving this pattern even further, please also review these [comments from the ISC Fall Summit 2017](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/63#pullrequestreview-67393334).
```
